examples/purchaseTesterTypescript/android/gradle.properties

```diff
@@ -34,7 +32,7 @@ reactNativeArchitectures=armeabi-v7a,arm64-v8a,x86,x86_64
 # your application. You should enable this flag either if you want
 # to write custom TurboModules/Fabric components OR use libraries that
 # are providing them.
-newArchEnabled=false
+newArchEnabled=true
```

Comment:
I believe this might be ok... I'm able to repro the issue with the footers not looking right in ios, which we should address when we can.

Reply:
I think this is better. If it works in new architecture, it will probably work in old architecture. Ideally we have a CI job at least that builds in the old arch? Not for this PR anyways